### PR TITLE
chore: add issue template

### PR DESCRIPTION
**Pull request by sboh1214**

### CODE REVIEW

**Review by yumincho — APPROVED** [review]
Approve!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+A clear and concise description of what the bug is.
+
+## 재현 순서
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+## 예상하는 올바른 결과
```
Comment (LarryKwon) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: Optional

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+A clear and concise description of what you expected to happen.
+
+## 에러 로그
```
Comment (LarryKwon) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: Optional

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- ```python --version```:
+- ```node --version```:
+
+## 테스트 환경
+
+- Device: [e.g. iPhone6]
+- OS: [e.g. iOS8.1]
+- Web Version: [e.g. 1.1.0]
+
+## 추가 정보
```
Comment (LarryKwon) [inline .github/ISSUE_TEMPLATE/bug_report.md:46]: Optional

**Review by LarryKwon — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+## 설명
+
+**Describe the solution you'd like.**
+
+A clear and concise description of what you want to happen.
+
+## 스크린샷
```
Comment (LarryKwon) [inline .github/ISSUE_TEMPLATE/feature_request.md:21]: Optional

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+If applicable, add screenshots to help explain your feature request.
+
+## 개발 환경
+
+- OS: [e.g. macOS]
+- ```python --version```:
+- ```node --version```:
+
+## 테스트 환경
```
Comment (LarryKwon) [inline .github/ISSUE_TEMPLATE/feature_request.md:31]: Optional

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+- Device: [e.g. iPhone6]
+- OS: [e.g. iOS8.1]
+- Web Version: [e.g. 1.1.0]
+
+## 추가 정보
```
Comment (LarryKwon) [inline .github/ISSUE_TEMPLATE/feature_request.md:39]: Optional

**Review by LarryKwon — CHANGES_REQUESTED** [review]

**Review by LarryKwon — APPROVED** [review]

**Pull request merged**